src/daidepp/keywords/press_keywords.py

```diff
@@ -8,19 +8,24 @@
 from daidepp.keywords.daide_object import _DAIDEObject
 
 if TYPE_CHECKING:
-    from typing import FrozenSet, Iterable, List, Optional, Set, Tuple, Union
+    from typing import Iterable, List, Optional, Tuple, Union
 
 
 @dataclass(eq=True, frozen=True)
 class PCE(_DAIDEObject):
-    powers: FrozenSet[Power]
+    powers: Tuple[Power]
 
     def __init__(self, *powers: Power):
-        object.__setattr__(self, "powers", frozenset(powers))
+        object.__setattr__(self, "powers", tuple(sorted(powers)))
```

Comment:
@aphedges To split the difference, could we do something like this? That way we're still ensuring that we have a set and just converting it to a tuple.

```suggestion
        object.__setattr__(self, "powers", tuple(sorted(frozenset(powers))))
```

Reply:
Is your goal to prevent a user from accidentally adding an element twice?

Reply:
@mjspeck, I'd appreciate a response when you have the time so I can make the changes you want.

Reply:
@mjspeck, I'd really appreciate as soon as possible. It's been over a week since I asked a question about your feedback, and I would like to have enough time to finish this PR and get the changes into our bot.

Reply:
I think we agreed to implement this change

Reply:
I implemented this in e8e705b32c86e5e4c9672fd0b7d3027028adcdb7.